### ISSUE TITLE
Context is not set in kubectl.

### ISSUE THREAD

**Issue by Spencer-Easton** [GoogleCloudPlatform/cloud-code-vscode#74]
**Environment**:
Extension version: 0.0.7
VSCode version: 1.33.1
OS: linux

**Description**:
Cloud Code: Deploy returns output 

> Context is not set in kubectl.

From the command line I see:
```
spencer@spencer-lappy:~$ kubectl config current-context
gke_ait-***-dev_us-central1-a_hello-world-server

```
I then even used command

> Search GKE clusers to set at current context

After following the prompts I got the response 
`Successfully set cluster "hello-world-server" as current context.`

I still got the error 

> Context is not set in kubectl.

**Comment by redborian** [comment]
Are you using multiple `.kubeconfig` files? Or, are you using custom KUBECONFIG environment variable?

**Comment by rsoutar** [comment]
Hi,

This might be a similar issue.

I had multiple clusters and projects on GCP. And when I choose which will be an active cluster. Somehow it still switch back to the old cluster while I do `Cloud Code: Deploy`.

My workaround is to remove context which I'm not going to use on the particular project. Then exit VSCode and open it again then run `Cloud Code: Deploy`

**Comment by redborian** [comment]
@sujit-kamireddy, can you see if you can add an option to deploy to current cluster instead of always switching to the previously deployed cluster?

**Comment by JCzz** [comment]
Any progress on this issue - I have the same?

**Comment by sujit-kamireddy** [comment]
@JCzz, we are working on the new deploy and debug flows and as part of that, users will be prompted if a context is not set in the kubeconfig upfront.

Meanwhile, can you check if a default context is set in the kubeconfig file?

**Comment by JCzz** [comment]
Thanks @sujit-kamireddy 

Sounds good, and yes my default context is set in the kubeconfig file. 
I can do: `skaffold run --default-repo=gcr.io/<projectid>/cloudcode`

As a workaround, that is what I will be using until the new deploy and debug flow. Looking forward to that :-)

**Comment by JCzz** [comment]
Hi @sujit-kamireddy

I got it to work:

1. Deleted Skaffold
2. Deleted .kube
3. gcloud container clusters get-credentials my-cluster

**Comment by sujit-kamireddy** [comment]
Thanks @JCzz for the workaround.

**Comment by JCzz** [comment]
Hmm, back to not-working. Any idea:

```
echo $KUBECONFIG
/Users/<computeruser>/.kube/config:/Users/< computeruser >/.kube/config_files/microconfig
```

**Comment by kschaab** [comment]
This was fixed in our 1.3.0 release!  Thanks for reporting this issue.
